xwiki-commons-tools/xwiki-commons-tool-verification-resources/src/test/java/org/xwiki/tool/checkstyle/UnstableAnnotationCheckTest.java

```diff
@@ -35,15 +32,15 @@
  * @version $Id$
  * @since 8.1M1
  */
-public class UnstableAnnotationCheckTest extends BaseCheckTestSupport
+public class UnstableAnnotationCheckTest extends AbstractModuleTestSupport
 {
     private DefaultConfiguration checkConfig;
 
     @Before
     public void setUp()
     {
         this.checkConfig = createCheckConfig(UnstableAnnotationCheck.class);
-        this.checkConfig.addAttribute("currentVersion", "8.1-SNAPSHOT");
+        this.checkConfig.addAttribute("currentVersion", "8.2-SNAPSHOT");
```

Comment:
@mflorea The only reason I changed it was because when the Checkstyle version was `8.0`, here `8.1-SNAPSHOT` was used for testing. I just thought it would be appropriate to make the change since I was updating the version to `8.1`.